cli/cmd/start/start.go

```diff
@@ -71,6 +71,7 @@ func StartCmd(ver string) *cobra.Command {
 				ConnectionCacheSize: 100,
 				MetastoreDriver:     "sqlite",
 				MetastoreDSN:        "file:rill?mode=memory&cache=shared",
+				QueryCacheSize:      1000,
```

Comment:
its 1K elements
1. I think it should be configurable via env var or property file, currently its not. 
2. I also think we should actually limit by memory instead of entries but that would require some more work. Marshalled results can return the size and we can use that to update the cache size and we can use on evict callback to deduct from the cache size.

Reply:
I agree, bounding by size would be ideal, its ok to start good defaults for element sizes, 1K seems low, maybe we can increase default to 100K entries.

Reply:
just assumed on higher side the size of results to be around 1 KB, so 1K elements would be around 1MB. 100K would be 100MB which seems on higher side for local setup not sure, we can keep 10K or 50K or 100MB is fine ?

Reply:
10K seems fine as well to begin with.

Reply:
10k sounds good to me as well.

Just a note that these are the hard-coded settings for running locally with the CLI. 

The entrypoint for a standalone runtime is `runtime/cmd/main.go` – there, all values should be configurable through env vars.

Reply:
done